contents/java/JavaReflections.md

```diff
@@ -15,6 +15,17 @@
 
 Authors: [Jeremy Goh](https://github.com/MightyCupcakes), [Yong Zhi Yuan](https://github.com/Zhiyuan-Amos)
 
+<box id="article-toc">
+
+* [Reflection‎](#reflection)
+  * [What is Java reflections‎](#what-is-java-reflections)
+  * [The Basics of Reflections‎](#the-basics-of-reflections)
+  * [Getting Started‎](#getting-started)
+  * [Applications‎](#applications)
```

Comment:
can add the subsection of applications to the side TOC and main TOC?

Reply:
Done!